docs/integrations/sources/height.md

```diff
@@ -0,0 +1,42 @@
+# Height
+
+This page contains the setup guide and reference information for the Height source connector.
+
+## Prerequisites
+
+To set up the Height source connector, you'll need the Height API key that you could see once you login and navigate to https://height.app/xxxxx/settings/api, and copy your secret key
+Website: https://height.app
+
+API Documentation: https://height.notion.site/API-documentation-643aea5bf01742de9232e5971cb4afda
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API secret which is copied from the settings page of height.app  |  |
+| `start_date` | `string` | Start date for   |  |
```

Comment:
@btkcodedev add the start date docs here, this sentence seems unfinished.

Reply:
Done